_includes/feed.liquid

```diff
@@ -3,7 +3,11 @@ layout: ./nodes.liquid
 section: smartContract
 ---
 
-<p>For implementation instructions, please refer to <a href="../get-the-latest-price/">Get the Latest Price</a>.</p>
+{% if metadata.solanafeeds %}
+    <p>For implementation instructions, please refer to <a href="/">Solana integration sample</a>.</p>
```

Comment:
```suggestion
    <p>The program address for this feed is publicly available in the <a href="https://explorer.solana.com/address/2yqG9bzKHD59MxD9q7ExLvnDhNycB3wkvKXFQSpBoiaE?cluster=devnet">Solana Devnet Explorer</a>.</p>
```

I will add a link to the implementation instructions later. For now, let's include the link to the program instead.

Reply:
Got it. Ok